packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -102,20 +102,22 @@ export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void
     }
 
     // Prune invalid SSZ objects every interval
-    const {persistInvalidSszObjectsDir} = args;
-    const pruneInvalidSSZObjectsInterval = persistInvalidSszObjectsDir
-      ? setInterval(() => {
-          try {
-            pruneOldFilesInDir(
-              persistInvalidSszObjectsDir,
-              (args.persistInvalidSszObjectsRetentionHours ?? DEFAULT_RETENTION_SSZ_OBJECTS_HOURS) * HOURS_TO_MS
-            );
-          } catch (e) {
-            logger.warn("Error pruning invalid SSZ objects", {persistInvalidSszObjectsDir}, e as Error);
-          }
-          // Run every ~1 hour
-        }, HOURS_TO_MS)
-      : null;
+    const {persistInvalidSszObjectsDir, persistInvalidSszObjects} = options.chain;
```

Comment:
I don't think this is correct, as `chain.persistInvalidSszObjectsDir` will always be `undefined`
https://github.com/ChainSafe/lodestar/blob/26e9bc4b15e8d04fe6af718934aee602da5d0a16/packages/cli/src/options/beaconNodeOptions/chain.ts#L46

see also 

https://github.com/ChainSafe/lodestar/blob/26e9bc4b15e8d04fe6af718934aee602da5d0a16/packages/cli/src/options/beaconNodeOptions/chain.ts#L12-L14

we might wanna look into consolidating those options more, in our ansible deployment we are not actually setting any `persistInvalidSszObjectsDir` which is why the pruning did not work but it won't work on this branch either as is

Reply:
it was tested and worked fine
`persistInvalidSszObjectsDir` always have value as defined in https://github.com/ChainSafe/lodestar/blob/4e86094453a34bb8cf9bad4224cb770fdf21bbdc/packages/cli/src/cmds/beacon/paths.ts#L43

then depending on `persistInvalidSszObjects` we'll set the interval or not

this is similar to the deployment, it all depends on `chain.persistInvalidSszObjectsDir`, we don't need to define `chain.persistInvalidSszObjectsDir`